jboss-kie-wildfly-common/added/launch/jboss-kie-wildfly-security-login-modules.sh

```diff
@@ -39,7 +40,11 @@ function build_login_module() {
     if [[ -n ${module} ]]; then
         login_module="<login-module code=\""${code}"\" flag=\""${flag}"\" module=\""${module}"\">"
     else
-        login_module="<login-module code=\""${code}"\" flag=\""${flag}"\">"
+        if [ "${flag}" != "optional" ]; then
```

Comment:
I don't see a reason to change it, flag will be set here: local login_module=$(build_login_module "LdapExtended" "flag")